### PR TITLE
use k8s.io/utils/env GetString get env

### PR DESCRIPTION
**Pull request by helen-frank**
Signed-off-by: helen <haitao.zhang@daocloud.io>

**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
use k8s.io/utils/env GetString get env
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/karmadactl/util/apiclient/apiclient.go [hunk 1]
```diff
@@ -30,12 +31,7 @@ var (
 // 3. In your home directory as ~/.kube/config
 func KubeConfigPath(kubeconfigPath string) string {
 	if kubeconfigPath == "" {
-		env := os.Getenv("KUBECONFIG")
-		if env != "" {
-			kubeconfigPath = env
-		} else {
-			kubeconfigPath = defaultKubeConfig
-		}
+		kubeconfigPath = env.GetString("KUBECONFIG", defaultKubeConfig)
```
Comment (RainbowMango) [inline pkg/karmadactl/util/apiclient/apiclient.go:34]: What's the benifit?

Comment (helen-frank) [reply pkg/karmadactl/util/apiclient/apiclient.go:34]: Code reuse

Comment (lonelyCZ) [reply pkg/karmadactl/util/apiclient/apiclient.go:34]: It seems to reduce codes of judging.

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**